public/locales/tr/translation.json

```diff
@@ -13,5 +13,128 @@
   },
   "ar": {
     "1": "Arapça"
```

Comment:
is this ``` "ar": {
    "1": "Arapça"	    "1": "Arapça"``` suposse to be here?!

Reply:
it is from before, at translation branch we will fix it all.